commands/download.js

```diff
@@ -1,14 +1,10 @@
 var chalk = require('chalk')
 var prettyBytes = require('pretty-bytes')
-var Dat = require('dat-js')
 var logger = require('status-logger')
 var speedometer = require('speedometer')
 var ui = require('../lib/ui')
 
-module.exports = function (args) {
-  if (args && args.exit) args.upload = false
```

Comment:
`args.upload = false` prevented re-seeding data. They are not really related... 

I think the thought with `--exit` was you want to download as quickly as possible, so uploading would slow that down.